tests/views/test_change_response_status.py

```diff
@@ -331,3 +356,43 @@ def test_not_started_status_is_present_for_no_longer_required(self, mock_request
         self.assertEqual(response.status_code, 200)
         self.assertIn(b"No longer required", data)
         self.assertIn(b"Not started", data)
+
+    @requests_mock.mock()
+    def test_not_started_status_is_present_for_completed(self, mock_request):
+        mock_request.post(url_sign_in_data, json={"access_token": self.access_token}, status_code=201)
+        mock_request.get(url_get_survey_by_short_name, json=survey)
+        mock_request.get(url_permission_url, json=user_permission_reporting_unit_edit_json, status_code=200)
+        self.client.post("/sign-in", follow_redirects=True, data={"username": "user", "password": "pass"})
+        mock_request.get(url_get_collection_exercises_by_survey, json=collection_exercise_list)
+        mock_request.get(url_get_business_by_ru_ref, json=business_reporting_unit)
+        mock_request.get(url_get_available_case_group_statuses, json=self.statuses)
+        mock_request.get(url_get_case_groups_by_business_party_id, json=case_groups_completed)
+        mock_request.get(url_get_case_by_case_group_id, json=[case])
+        mock_request.get(url_get_case_events, json=case_events)
+        mock_request.get(get_respondent_by_id_url, json=respondent)
+
+        response = self.client.get(f"/case/{ru_ref}/response-status?survey={short_name}&period={period}")
+
+        data = response.data
+        self.assertEqual(response.status_code, 200)
+        self.assertIn(b"Completed", data)
+        self.assertIn(b"Not started", data)
+
+    @requests_mock.mock()
+    def test_not_started_status_is_not_present_for_not_started_without_permission(self, mock_request):
```

Comment:
I find this name difficult to understand, specially with 2 not_started in it

Reply:
Updated to be more clear